Emby.Server.Implementations/Services/ServicePath.cs

```diff
@@ -538,7 +509,7 @@ public object CreateRequest(string pathInfo, Dictionary<string, string> queryStr
                     pathIx++;
                 }
 
-                requestKeyValuesMap[propertyNameOnRequest] = value;
+                requestKeyValuesMap[variableName] = value;
```

Comment:
won't this be a problem that previously `requestKeyValuesMap` had lower-case keys, and now they could be arbitrary-cased?

Reply:
No, I changed the dict to ignorecase.

Reply:
I mean, someone might rely on serving _out_ those keys, and they were served lower-case before while now they're arbitrary. It's a minor concern anyway...